src/directives/recenter.js

```diff
@@ -0,0 +1,38 @@
+goog.provide('ngeo.recenterDirective');
+
+goog.require('ngeo');
+
+
+/**
+ * @ngdoc directive
+ * @ngname ngeoRecenter
+ */
+ngeo.recenterDirective = function() {
+  return {
+    restrict: 'A',
+    link: function($scope, $element, $attrs) {
+      var mapExpr = $attrs['ngeoRecenterMap'];
+      var map = /** @type {ol.Map} */ ($scope.$eval(mapExpr));
+
+      function recenter(element) {
+        var extent = element.attr('extent');
+        if (extent !== undefined) {
+          map.getView().fit(JSON.parse(extent), map.getSize());
```

Comment:
You're right but the use case I was thinking is an extent as a scope variable; something like:
```html
<a href="#" extent="mainCtrl.extents.europa" translate>Europa</a>
```